### ISSUE TITLE
Interactive physics shape editor

### ISSUE THREAD

**Issue by karl-ourmachinery** [OurMachinery/themachinery-public#386]
When editing the physics shape, for example a box shape, I'd like to edit the extends of it by clicking and dragging the boundary lines in the scene view.

Also, it would be nice if the shape had a local position of it's own, so I don't have to put the shape under a separate entity to offset the shape from entity origin.

**Comment by frank-ourmachinery** [comment]
The volume component has dragging functionality, do we want to reuse this for physics shapes?

**Comment by niklas-ourmachinery** [comment]
Yes, I think the physics shape component should have similar dragging functionality... and the sculpt component too.

Yes, having a local position or "pivot" for the shape would be helpful too and reduce the number of "extra" entities we need to create.

**Comment by karl-ourmachinery** [comment]
Yeah, the local position for the shape would be great. And PhysX supports this for its internal representation of collision shapes, so its no problem.

**Comment by karl-ourmachinery** [comment]
(put the position stuff in a separate issue https://github.com/OurMachinery/themachinery-public/issues/446)
